cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java

```diff
@@ -84,6 +86,21 @@ public synchronized String toString() {
     return sb.toString();
   }
 
+  public synchronized Map<String, Map<String, String>> getJsonStructure() {
+    Map<String, Map<String, String>> progressMap = new LinkedHashMap<>();
+    for (int i = 0; i < _steps.size(); i++) {
+      OperationStep step = _steps.get(i);
+      long time = (i == _steps.size() - 1 ? System.currentTimeMillis() : _startTimes.get(i + 1)) - _startTimes.get(i);
+      Map<String, String> stepProgressMap = new LinkedHashMap<>();
```

Comment:
Why do we use `LinkedHashMap` and not a `HashMap` here?

Reply:
because I also want to main the order of 
description, time, percentage
in each step